flask_annex/s3.py

```diff
@@ -10,6 +12,11 @@
 
 MISSING = object()
 
+
+def defined(obj):
```

Comment:
nit: more a nit--this might be *slightly* more clear

```suggestion
def is_defined(obj):
```

Reply:
@c0state changed